client/state/test/index.js

```diff
@@ -0,0 +1,47 @@
+require( 'lib/react-test-env-setup' )();
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import mockery from 'mockery';
+
+describe( 'state', () => {
+	var createReduxStore;
+	before( () => {
+		mockery.registerMock( 'lib/themes/middlewares.js', {
```

Comment:
It doesn't affect these tests, but I think this should be `client/themes/middlewares.js`.